keep-web/ui/src/app.css

```
@@ -0,0 +1,164 @@
+:root {
+  color-scheme: dark;
+  --bg: #14181f;
+  --panel: #1c222c;
+  --border: #2a313d;
+  --text: #e6e9ef;
+  --muted: #8b93a1;
+  --accent: #0877d2;
+  --ok: #3fb950;
+  --no: #e5534b;
+  font-family: system-ui, -apple-system, sans-serif;
```

Stylelint (17.11.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)